src/components/m-table-cell.js

```diff
@@ -36,23 +37,23 @@ export default class MTableCell extends React.Component {
       if (this.props.value instanceof Date) {
         return this.props.value.toLocaleDateString();
       } else if(isoDateRegex.exec(this.props.value)) {
```

Comment:
Can you also add dateLocale here as seen here: parseISO(this.props.value).toLocaleTimeString(dateLocale);

Reply:
@Domino987 assuming you meant on line 40, I've just pushed that change.